cameraserver/src/main/native/cpp/cameraserver/CameraServer.cpp

```diff
@@ -56,21 +56,21 @@ static wpi::StringRef MakeSourceValue(CS_Source source,
   CS_Status status = 0;
   buf.clear();
   switch (cs::GetSourceKind(source, &status)) {
-    case cs::VideoSource::kUsb: {
+    case CS_SourceKind::CS_SOURCE_USB: {
```

Comment:
If we're going to use the CS_SourceKind enum values, we don't need to scope them at all (it's a C enum, not a C++ enum class).